pkg/service_sync/messaging.go

```diff
@@ -1,6 +1,9 @@
 package service_sync
 
 import (
+	"fmt"
+	vanClient "github.com/skupperproject/skupper/client"
```

Comment:
One concern I have with tying `service_sync` with `VanClient` is that on Podman
sites we won't have the event recorder available. So what if we have an
event logger interface and, in this case, the kubernetes implementation uses the
event recorder, but let it open for other platforms to use what suits them better?

This is not a problem for this PR, but something to discuss together considering that
podman is already in place (although service sync is not yet implemented for podman)
and eventually other platforms will be supported soon.

Maybe an EventLogger interface could be added under `pkg/domain` ?